doc/en/user/source/styling/mbstyle/cookbook/artifacts/mbstyle_line_dashedline.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": 8,
+  "name": "simple-dashedline",
+	"layers":[
```

Comment:
Use spaces instead of tabs here.

Most of the subsequent styles have the same problem, and need to be fixed as well.